docs/titan-reference/button-reference.md

```diff
@@ -0,0 +1,485 @@
+
+\<Avo\> + \<Clear\> with a plus symbol means to hold down the first button while pressing the second button.
+
+{Select} means to press a playback select button.
+
+---
+
+ ## \<@\> 
+ (by numeric keypad) Allows numeric entry of attribute values. For the \<@\> buttons next to the wheels see [Wheel @](#wheel-).
+
+ Example: \<Fixture\> 1 \<And\> 5 \<And\> 7 \<@\> 75 \<Enter\>
+
+  Buttons | Action
+ --------|--------
+ \<@\> ``<n>`` \<Enter\> | Set intensity of currently selected fixtures to ``<n>`` in the programmer. (``<n>`` normally 0-99, optionally can be single digit 0-9, see [Formatting](../system-settings/user-settings.md#formatting))
+ \<@\> \<@\> | Set intensity of currently selected fixtures to full in the programmer.
+  
+ --- 
+## \<Align\> 
+ (D9 only) Allows copying of attributes between fixtures. On consoles other than D9 this function is accessed using \<ML Menu\>. See [Align Fixtures](../controlling-fixtures/changing-fixture-attributes.md/#align-fixtures).
+
+ --- 
+## \<All\> 
+ Used to start and end odd/even selection from a range of selected fixtures. See [Selecting fixtures using a pattern](../controlling-fixtures.md/#selecting-fixtures-using-a-pattern).
+
+ --- 
+## \<And\> 
+ Used when selecting multiple items using the numeric keypad, can be combined with \<Through\> and \<Not\>. See [Selecting fixtures and dimmers by number](../controlling-fixtures.md/#selecting-fixtures-and-dimmers-by-number-channel). Some consoles do not have this button, an \[And\] softkey is also provided.
+
+ Example:  \<Fixture\> 1 \<And\> 5 \<And\> 7
+
+ --- 
+## \<Assign\> 
+ (D9 and ST only) Allows you to reassign attributes controlled by the trackball and (on D9 only) intensity wheel. See [Trackball](../controlling-fixtures/changing-fixture-attributes.md#trackball-diamond-9-and-sapphire-touch-only).
+
+ --- 
+## \<Avo\> 
+ Acts as a "Shift" button when held down, giving access to different functions. 
+
+  Buttons | Action
+ --------|--------
+ \<Avo\> + \<All\> | Set zero intensity into programmer for unselected fixtures (same as \<Rem Dim\>), see [Turn off unselected fixtures](../controlling-fixtures.md#turn-off-unselected-fixtures-remainder-dim). Also used when setting Attribute Times to deselect all fixtures, see [Attribute Fade Times](../cue-lists/cue-list-timing.md#individual-attribute-fade-times).
+ \<Avo\> + \<Close\> | Close all workspace windows, see [Workspace windows](../titan-basics/workspace-windows.md#changing-window-size-and-appearance).
+ \<Avo\> + \<Copy\> | Same as pressing \<Move\> (for consoles which don't have a Move button, but works on all consoles)
+ \<Avo\> + \<Disk\> | Enter System menu to set up the console, see [System Menu](../system-settings/the-system-menu.md).
+ \<Avo\> + \<Exit\> | Exit menu structure back to the top level
+ \<Avo\> + \<Group\> {Select} | Record group to handle, see [Record a group](../controlling-fixtures/fixture-groups.md#record-a-group).
```

Comment:
This works for any noun, not just groups: also Palette, Playback (D9), Cue, Macro (not fixture)